### PR TITLE
[2448] Amended guidance text

### PR DESCRIPTION
**Pull request by defong**
### Context
Guidance text

### Changes proposed in this pull request
Amended guidance text

#### Before

image

#### After

image

#### Before

image

#### After

image

#### Before

image

#### After

image

### Checklist

- [x] Make sure all information from the Trello card is in here
- [x] Attach to Trello card
- [x] Rebased master
- [x] Cleaned commit history
- [x] Tested by running locally

### CODE REVIEW

**Comment by defong** [comment]
[Add new guidance text](https://trello.com/c/aH4i0UaZ/2448-add-new-guidance-text)

**Comment by defong** [comment]
[Add new guidance text](https://trello.com/c/aH4i0UaZ/2448-add-new-guidance-text)

**Comment by defong** [comment]
### Updated screenshots
image

image

image

**Comment by defong** [comment]
image

File: app/views/courses/about.html.erb [hunk 1]
```diff
@@ -22,21 +22,27 @@
...
+        <li>league-table rankings and student employability ratings</li>
+        <li>quotes from past students</li>
       </ul>
 
-      <p class="govuk-body">Don’t repeat the same description for different courses. If you’re copying content you should tailor it to fit this course.</p>
+      <p class="govuk-body">Remember to:</p>
+      <ul class="govuk-list govuk-list--bullet">
+        <li>make your content specific to this course - don’t repeat the descriptions you’ve used for other courses</li>
+        <li>use short paragraphs (no more than five sentences each)</li>
+        <li>use bullet points where possible (use Preview to check they’re correctly formatted)</li>
```
[the remark this answers is not in the document]
Comment (dankmitchell) [reply app/views/courses/about.html.erb]: Deffo a typo. Could even warrant a rejection. What do you think @TobyRet :trollface:

[the remark this answers is not in the document]
Comment (defong) [reply app/views/courses/about.html.erb]: literally taking the `p` now

**Pull request merged**